.github/workflows/fossa.yml

```
@@ -7,7 +7,7 @@ on:
       - 'dependabot/**'
 
 permissions:
-  contents: read # Required by actions/checkout@v4 to fetch the repository contents.
+  contents: read # Required by actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7 to fetch the repository contents.
```

```suggestion
  contents: read # Required by actions/checkout to fetch the repository contents.
```
We can remove the version information from the comments to reduce maintenance costs.